### PR TITLE
Change the descriptions of the schemas in UpgradePathFinder to make…

### PR DESCRIPTION
**Pull request by jonknight73**
…them clearer

### CODE REVIEW

File: morf-core/src/main/java/org/alfasoftware/morf/metadata/SchemaHomology.java [hunk 1]
```diff
@@ -384,7 +384,7 @@ private List<String> toUpperCase(List<String> strings) {
    */
   private void matches(String description, Object value1, Object value2) {
     if (ObjectUtils.notEqual(value1, value2)) {
-      difference(String.format("%s does not match: [%s] in %s, [%s] in %s", description, value1, schema1Name, value2, schema2Name));
+      difference(String.format("%s does not match: [%s] in [%s], [%s] in [%s]", description, value1, schema1Name, value2, schema2Name));
     }
   }
```
Comment (jsimlo) [inline morf-core/src/main/java/org/alfasoftware/morf/metadata/SchemaHomology.java:389]: Changes in this file are good, let's merge them.

File: morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java [hunk 2]
```diff
@@ -109,13 +109,13 @@ public SchemaChangeSequence determinePath(Schema current, Schema target, Collect
...
+    if (!schemasMatch(target, trialUpgradedSchema, "target domain schema (e.g. schema from Java source)", "database or xml schema with upgrades applied", exceptionRegexes)) {
       throw new NoUpgradePathExistsException();
     }
 
     // Now reverse-apply those changes to see whether they get us back to where we started
     Schema reversal = schemaChangeSequence.applyInReverseToSchema(trialUpgradedSchema);
-    if (!schemasMatch(reversal, current, "upgraded schema reversal", "current schema", exceptionRegexes)) {
+    if (!schemasMatch(reversal, current, "upgraded database or xml schema with upgrades reversed", "current database or xml schema", exceptionRegexes)) {
       throw new IllegalStateException("Upgrade reversals are invalid");
     }
```
Comment (jsimlo) [inline morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java:120]: Let's revert these changes and wait for the consensus outcome of recently revived discussions.

File: morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java [hunk 3]
```diff
@@ -199,20 +199,20 @@ private Map<java.util.UUID, CandidateStep> candidateStepsByUUID() {
...
       public void difference(String message) {
         log.info(message);
       }
     };
 
-    SchemaHomology homology = new SchemaHomology(differenceWriter, firstSchemaContext, secondScehmaContext );
+    SchemaHomology homology = new SchemaHomology(differenceWriter, firstSchemaContext, secondSchemaContext );
     if (homology.schemasMatch(targetSchema, trialSchema, exceptionRegexes)) {
       log.info("Schemas match");
       return true;
```
Comment (jsimlo) [inline morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java:218]: These fixes are good, let's merge them.

**Review by jsimlo — CHANGES_REQUESTED** [review]

File: morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java (hunk 3 above)
Comment (harrisric) [reply morf-core/src/main/java/org/alfasoftware/morf/upgrade/UpgradePathFinder.java:218]: Superceded by #261 ?

**Comment by jonknight73** [comment]
Actioned on #261

**Pull request closed without merging**